airflow/kubernetes/kube_client.py

```diff
@@ -41,6 +42,17 @@ def _load_kube_config(in_cluster, cluster_context, config_file):
 def get_kube_client(in_cluster=conf.getboolean('kubernetes', 'in_cluster'),
                     cluster_context=None,
                     config_file=None):
+    """
+    Retrieves Kubernetes client
```

Comment:
```suggestion
    Retrieves Kubernetes client
    
```
New line is still needed

Reply:
Done.